### PR TITLE
Update active committers and support white papers part

### PR DESCRIPTION
**Pull request by christophechevalier**

### CODE REVIEW

File: community.md
Comment (cdeneux) [inline community.md]: Perhaps can you add the role of everyone inside the Petals team ?

File: support.md
Comment (cdeneux) [inline support.md]: "And some video conferences are also available"

**Review by cdeneux — CHANGES_REQUESTED** [review]

File: community.md
Comment (vincent-zurczak) [reply community.md]: Why do you add `target: _blank`?
And roles can evolve. This is useless.

File: support.md
Comment (vincent-zurczak) [reply support.md]: "And some video conferences are available".
Avoid and + also together. :)

**Review by vincent-zurczak — CHANGES_REQUESTED** [review]
The use of `target: _blank` is useless.
If people want to open links in another tab or window, they will be able to do it. No need to force the opening somewhere else.

File: community.md
Comment (christophechevalier) [reply community.md]: This will open it in a new tab. I find it more practical when it comes to opening an external link.  
I didn't push roles ?!

**Review by vincent-zurczak — APPROVED** [review]

**Comment by vincent-zurczak** [comment]
OK for me. :+1: 
@cdeneux Waiting for your approval. If you merge, please select **squash and merge** in the drop-down list.

**Review by cdeneux — APPROVED** [review]

**Pull request merged**
